augur/curate/parse_genbank_location.py

```diff
@@ -21,8 +21,8 @@ def parse_location(
     #
     # See GenBank docs for their "country" field:
     # https://www.ncbi.nlm.nih.gov/genbank/collab/country/
-    location_field = record.get(location_field_name, "")
-    if not location_field:
+    location_field = record.get(location_field_name, None)
+    if location_field == None:
```

Comment:
_non-blocking nit_

Check for singletons like None are usually done with `is`

```suggestion
    if location_field is None:
```

Reply:
force-pushed a correction.